Packs/PhishLabs/Integrations/PhishLabsIOC_EIR/PhishLabsIOC_EIR.py

```diff
@@ -329,40 +371,81 @@ def fetch_incidents_command(
     incidents_raw: List = []
     # Set last run time
     occurred_format = '%Y-%m-%dT%H:%M:%SZ'
+    subcategories = [cat.replace(" ", "") for cat in subcategories]
     if not last_run:
         datetime_new_last_run, _ = parse_date_range(date_range=fetch_time,
                                                     date_format=occurred_format)
     else:
         datetime_new_last_run = last_run
     # Query incidents by limit and creation time
-    total = 0
-    offset = 50
+    offset = 0
     limit_incidents = int(limit)
     limit_page = min(50, limit_incidents)
-    raw_response = client.get_incidents(created_after=datetime_new_last_run,
-                                        offset=offset,
-                                        limit=limit_page,
-                                        sort='created_at',
-                                        direction='asc')
-    while raw_response.get('metadata', {}).get('count') and total < limit_incidents:
-        raws.append(raw_response)
-        incidents_raw += raw_response.get('incidents', [])
-        total += int(raw_response.get('metadata', {}).get('count'))
-        offset += int(raw_response.get('metadata', {}).get('count'))
-        if total >= limit_incidents:
-            break
-        if limit_incidents - total < 50:
-            limit_page = limit_incidents - total
-        raw_response = client.get_incidents(offset=offset,
-                                            created_after=datetime_new_last_run,
-                                            limit=limit_page,
-                                            sort='created_at',
-                                            direction='asc')
+    # Fetch open Phishlabs incidents
+    raws, incidents_raw = fetch_incidents_per_status(client=client,
```

Comment:
is this a change that existing customers might not expect, or an actual bug fix? let's continue discussing this on slack